frontend/src/components/search/SearchBar.jsx

```diff
@@ -0,0 +1,47 @@
+// NPM Packages
+import React, { useEffect, useState } from "react";
+import { useHistory } from "react-router-dom";
+import UserApi from "../../api/UserApi";
```

Comment:
Pls could you add an empty line after line 3 and then add `// Project files` above the `UserApi` import?